frontend/src/features/help/components/text/DetailNavigationHelpText.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
Decided to structure help text as components to provide the desired text formatting. Probably possible to do some html string formatting but I don't see a downside to doing things this way.